bindings/python/compile_flags.in

```diff
@@ -1 +1 @@
--I@top_srcdir@/include @COMPILETIME_OPTIONS@ @CPPFLAGS@ @BOOST_CPPFLAGS@ @OPENSSL_INCLUDES@
+-std=c++11 -I@top_srcdir@/include @COMPILETIME_OPTIONS@ @CPPFLAGS@ @BOOST_CPPFLAGS@ @OPENSSL_INCLUDES@
```

Comment:
This seems very GCC centric. I was under the impression that the whole point of the c++11 m4 scripts was to not make these kinds of assumptions. I found though that that script adds -std=c++11 to the CXX variable, which seems like a poor choice.